### PR TITLE
Fix `ONCE` environment variable ignorance

### PR DESCRIPTION
**Pull request by dieselburner**
Hi there,

Here's the MR to [fix the issue I have faced](https://github.com/virtualzone/compose-updater/issues/12) with `ONCE` environment variable being ignored, when executed via Docker.

One important note: this changes `docker-compose-watcher` (or, in fact, `compose-updater`) default value of `once` to false, so that it would be possible to remove `-once=0` from [Dockerfile](https://github.com/virtualzone/compose-updater/blob/master/Dockerfile#L37), but still keeping the old behavior:

```Dockerfile
CMD ["docker-compose-watcher", "-once=0", "-printSettings"]
```

The reason of removing this argument from `CMD` above is rather simple: due to code structure command line arguments takes precedence over environment variables, which I think is the right way to go. However, keeping `-once=0` will prevent ability to override behavior of passing environment variable `ONCE` to Docker.

Basically, this MR makes everyone happy.

Few other small things irrelevant to original issue comes as a bonus.

### CODE REVIEW

**Comment by dieselburner** [comment]
@virtualzone I would appreciate if you could review this and merge, or provide comments what needs to be addressed.

**Comment by dieselburner** [comment]
@virtualzone, it's been more than a year now since my last request. I would appreciate if you could have a quick look. Thanks.

**Comment by limdingwen** [comment]
Hi, I have merged this into my fork. Please see https://github.com/limdingwen/compose-updater/pull/2
